### ISSUE TITLE
Title attributes on content editor.

### ISSUE THREAD

**Issue by Garbee** [wardrobecms/core-archived#33]
The title attributes to the content editor buttons are irrelevant and/or missing completely. It would be nice if each button was given a title which is the action of the button. This improves a11y and reduces ambiguity as to what the buttons do.

**Comment by ericlbarnes** [comment]
Here is what I am seeing is this not the case for you? 

![screen shot 2013-09-01 at 11 22 50 pm](https://f.cloud.github.com/assets/116662/1065248/00e7b4d2-137f-11e3-9500-66ab7b437fa8.png)

Maybe I am not following correct?

**Comment by spamoom** [comment]
I think what @Garbee means is it should say "Link (⌘-K)"

**Comment by Garbee** [comment]
(sorry for response delay, crazy few days just happened.) Well, honestly I don't think the keyboard shortcuts should be shown at all. For instance I had Ctrl + Shift + L being shown for one of my shortcuts on Ubuntu, however that locks the screen. So it isn't useful at all.

The title attributes should just be, "Bold", "Italics", "Blockquote", "Unordered List", "Ordered List", "Link", "Image", "Edit as code", "Undo", "Redo", "Tag post", "Publish date". The keyboard shortcuts are an enhancement to access the buttons, not the primary way to interact with them nor what the buttons stand for.

**Comment by ericlbarnes** [comment]
Thanks guys. I will get this updated.

**Comment by ericlbarnes** [comment]
Pushing this to the v1.1 milestone as it's more involved than I originally thought. These are defined pretty deep in the editor and at first glance doesn't seem to be easily over-ridable.

**Comment by Garbee** [comment]
Looks like it has to do with [this](https://github.com/wardrobecms/core/blob/master/assets/vendor/plugins/editor/editor.js#L6805-L6809) area of the code. Setting the title attribute equal to the shortcut when it really should (imo) be set to the name instead.

**Comment by Garbee** [comment]
I just updated line 13950 of public/packages/wardrobe/core/admin/js/structure.js (and removed the following keyboard config lines) to be `el.title = name;` and then refreshed the backend. Those that had names setup work fine, however there are still the "Edit as code" buttons and later that don't get a title attached.

I'm not sure how to override the editors toolbar creation function so we can set custom titles without touching the core code. I'll look into it over the next few days though.

**Comment by ericlbarnes** [comment]
I was short on time tonight as I'm about to go on vacation. Going off memory the setup is in post_view.coffee. 

```
—
```

Eric Barnes

**Comment by rtablada** [comment]
I'll check it out a bit. Not sure if the tool tips are customizable while also keeping the functioning keyboard shortcuts...
